velox/functions/sparksql/ToJson.h

```diff
@@ -0,0 +1,400 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <expression/ComplexViewTypes.h>
+#include <functions/lib/DateTimeFormatter.h>
+#include <functions/lib/TimeUtils.h>
+#include <velox/type/DecimalUtil.h>
```

Comment:
To include Velox headers, we usually start with `velox/...` and use `""`. Perhaps refer to the examples in the other file.